pyopia/dataexport/ecotaxa.py

```diff
@@ -0,0 +1,233 @@
+"""
+Create data bundle from PyOPIA stats that can be imported into EcoTaxa.
+
+EcoTaxa: https://ecotaxa.obs-vlfr.fr/
+"""
+
+import numpy as np
+import pyopia
+import pyopia.io
+import pyopia.statistics
+import zipfile
+import io
+import xarray as xr
+import pandas as pd
+import matplotlib.pyplot as plt
+from typing import Optional
+from pathlib import Path
+from tqdm.rich import tqdm
+
+_ecotaxa_dict = {
+    "img_file_name": ("export_name", "str"),  # source: statsrow
+    "img_rank": ("img_rank", "float"),  # source: we will make it exist
+    "object_id": ("export_name", "str"),  # source: statsrow
+    "object_lat": (
+        "latitude",
+        "str",
+    ),  # needs to be in decimal degrees, source: auxillary data
+    "object_lon": (
+        "longitude",
+        "str",
+    ),  # needs to be in decimal degrees, source: auxillary data
+    "object_date": (
+        "timestamp",
+        "float",
+    ),  # UTC, format: YYYYMMDD, source: statsrow timestamp.dt.strftime(%Y%m%d)
+    "object_time": (
+        "timestamp",
+        "float",
+    ),  # UTC, format: HHMMSS, source: statsrow timestamp.dt.strftime(%H%M%S)
+    "object_depth_min": ("depth", "float"),  # source: statsrow
+    "object_depth_max": ("depth", "float"),  # source: statsrow
+    "object_major": ("major_axis_length", "float"),  # source: statsrow
+    "object_minor": ("minor_axis_length", "float"),  # source: statsrow
+    "object_circ.": ("equivalent_diameter", "float"),  # source: statsrow
+    "process_id": (
+        "process_id",
+        "float",
+    ),  # if missing will be added by EcoTaxa
+    "process_img_software_version": ("pyopia.__version__", "str"),
+    "process_img_resolution": ("process_img_resolution", "float"),
+    "process_particle_pixel_size_?m": (
+        "pixel_size",
+        "float",
+    ),  # needs to be converted from pixels to micrometers, the ? is a choice of EcoTaxa
+    "process_date": (
+        "process_date",
+        "float",
+    ),  # we will make it exists, datetime.today().strftime('%Y%m%d'),
```

Comment:
Typo in the comment, and also same question as above about date, float and string